lib/cos.js

```diff
@@ -0,0 +1,134 @@
+
+exports.initialize = function initializeDataSource(dataSource, callback) {
+	//console.log(dataSource.settings);
```

Comment:
Would you mind using the LoopBack style for formatting and indentation? Just run `npm run lint:fix` to apply our conventions.